### ISSUE TITLE
Works on chrome

### ISSUE THREAD

**Issue by bcrhbrhcdb** [Sh1N02/blooket-cheats#2]
i recommend you add back the 'auto sell dupes' feature

**Comment by Sh1N02** [comment]
What did you mean?
This one? 
![2D5ADFFE-D766-4DB8-90FA-4781CB9A48D2](https://github.com/Sh1N02/blooket-cheats/assets/160599599/46adb84a-2559-40b6-b62a-d4818df37252)

**Comment by bcrhbrhcdb** [comment]
No, when you buy something and its a dupe of a blooket you own, it should sell it.

**Comment by Sh1N02** [comment]
> No, when you buy something and its a dupe of a blooket you own, it should sell it.

I just saw this sorry
That option doesn’t exist, I can’t find it anywhere

**Comment by Githubman6996** [comment]
> i recommend you add back the 'auto sell dupes' feature

It broke that's why it's gone
